docs/source/pages/using.rst

```diff
@@ -233,13 +282,35 @@ Now we can run the policy engine over these results and enforce a policy:
 
 .. code-block:: shell
 
-  ./run_macaron.sh verify-policy -o outputs -d outputs/macaron.db --file oci-micronaut.dl
+  ./run_macaron.sh verify-policy -o outputs -d outputs/macaron.db --file <policy_file>
+
+In this example, the Datalog policy files for both ways (as mentioned previously) are provided in `oci-micronaut-repo.dl <../_static/examples/oracle-quickstart/oci-micronaut/policies/oci-micronaut-repo.dl>`__ and `oci-micronaut-purl.dl <../_static/examples/oracle-quickstart/oci-micronaut/policies/oci-micronaut-purl.dl>`__.
+
+The differences between the two policy files can be observed below:
+
+.. tabs::
+
+  .. code-tab:: prolog Using repository complete name
+
+    apply_policy_to("oci_micronaut_dependencies", repo_id) :- is_repo(repo_id, "github.com/oracle-quickstart/oci-micronaut", _).
+
+  .. code-tab:: prolog Using PURL string
+
+    apply_policy_to("oci_micronaut_dependencies", component_id) :- is_component(component_id, "<target_software_component_purl>").
+
+The PURL string for the target software component can be obtained when the :ref:`analyze command <analyze-action>` has finished. For example:
```

Comment:
```suggestion
The PURL string for the target software component is printed to the console by the :ref:`analyze command <analyze-action>`. For example:
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/229e0394f00c7f257f46987c5549b0d72143aa2a